### ISSUE TITLE
Get pachyderm homebrew recipe added to homebrew core library

### ISSUE THREAD

**Issue by sjezewski** [pachyderm/pachyderm#277]
They don't often do it for self-author-requests, but if we do want this, we have some homework to do:
- make sure our formula passes all their audit tools
- they don't like scripts that download deps (so .... no `go get` ... basically)
- other go based tools explicitly list all dependencies (e.g. https://github.com/Homebrew/homebrew-core/blob/master/Formula/consul.rb#L23)
- we could do this w a script from a vendor file ... but today we don't have one for client
- if we have to trade vendoring client for homebrew core-lib support ... I'm not sure we're willing to make the trade

Basically it would enable:

```
brew install pachctl
```

Instead of:

```
brew tap pachyderm/tap && brew install pachctl
```

**Comment by sjezewski** [comment]
This post suggests there's some tooling for this:

http://stackoverflow.com/questions/27097005/how-to-package-go-project-for-homebrew

But the link moved, and I only see the explicit list of resources :-/

**Comment by sjezewski** [comment]
Aha, here it is:

https://github.com/samertm/homebrew-go-resources

**Comment by sjezewski** [comment]
Dupe of https://github.com/pachyderm/pachyderm/issues/598
